test/odbc/psqlodbc/test_pg_varchar.cpp

```diff
@@ -0,0 +1,655 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "psqlodbc_tests_common.h"
+#include <string.h>
+#include <iostream>
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.varchar_table_odbc_test";
+const string VIEW_NAME = "master_dbo.varchar_view_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "varchar_1";
+const string COL3_NAME = "varchar_8000";
+const string COL4_NAME = "varchar_20";
```

Comment:
We don't need 4 different names for each of these. Can just use the same one.
This would also reduce the table constraints you need to make when testing the primary key, composite key, and unique key constraints